packages/pwa/app/components/_error/index.jsx

```diff
@@ -15,66 +28,71 @@ import {Helmet} from 'react-helmet'
 // 3. A routed component throws an error in `render()`.
 //
 // It must not throw an error. Keep it as simple as possible.
-const Error = (props) => {
-    const {stack, status} = props
 
-    const isNotFoundError = status === 404
-    const title = isNotFoundError
-        ? 'Sorry, we cannot find this page 😓'
-        : 'Sorry, there is a problem with this page 😔'
-    const message = 'To continue, go back home.'
+const Error = (props) => {
+    const {stack} = props
+    const history = useHistory()
+    const styles = useMultiStyleConfig('Error')
 
+    const title = "This page isn't working"
     return (
-        <div id="app" className="t-error">
+        <Flex id="sf-app" {...styles.container} flexDirection="column">
             <Helmet>
                 <title>{title}</title>
             </Helmet>
-
-            <div className="t-error__header">
-                <a href="/">
-                    <img
-                        src="https://www.mobify.com/wp-content/uploads/logo-mobify-white.png"
-                        alt="Mobify Logo"
-                        height="35"
+            <Box as="header" {...styles.header}>
+                <Box {...styles.headerContent}>
+                    <IconButton
+                        aria-label="logo"
+                        icon={<BrandLogo {...styles.logo} />}
+                        {...styles.icons}
+                        variant="unstyled"
+                        onClick={() => history.push('/')}
                     />
-                </a>
-            </div>
-
-            <main id="app-main" role="main">
-                <div className="t-error__container">
-                    <div className="t-error__container-flexbox">
-                        <div className="t-error__container-cloud u-margin-top-lg u-margin-bottom-lg">
-                            <svg
-                                aria-hidden="true"
-                                className="t-error__cloud-icon"
-                                title="Error Cloud"
-                                aria-labelledby="icon-1-1"
+                </Box>
+            </Box>
+            <Box as="main" id="app-main" role="main" flex={1} layerStyle="page" {...styles.main}>
+                <Flex {...styles.content} flexDirection="column" justify="center">
+                    <Flex align="center" mb={20} flexDirection="column">
```

Comment:
There seems to be a lot of inline stying as well as a theme file for this component. What is the logic you used to determine whether the style was in the theme or inlined? I would have assumed that it was all in the theme if you went out of your way to create one.